bin/runtime-common/src/messages_generation.rs

```diff
@@ -95,13 +98,32 @@ where
 				.expect("TrieMut::insert should not fail in benchmarks");
 			storage_keys.push(storage_key);
 		}
+
+		if add_duplicate_key {
```

Comment:
Probably worth add test for [that](https://github.com/paritytech/parity-bridges-common/pull/2155#discussion_r1205446525) here too

Reply:
I mean test for extra trie node. But NVM - having test in the `vec_db.rs` is enough